docs/community/governance.md

```diff
@@ -94,7 +94,7 @@ For more information on Repository Maintainers--their responsibilities, who they
 
 Area Experts are people with knowledge of specific components or technologies in the PowerShell domain. They are responsible for code reviews, issue triage, and providing their expertise to others.
 
-They have [write access](https://help.github.com/articles/permission-levels-for-an-organization-repository/) to the PowerShell repository which gives them the power to:
+They have [write access](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/managing-access-to-your-organizations-repositories) to the PowerShell repository which gives them the power to:
```

Comment:
```suggestion
They have [write access](https://docs.github.com/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/managing-access-to-your-organizations-repositories) to the PowerShell repository which gives them the power to:
```

Reply:
I had it like this because it just redirects to the country of origin (or browser?)'s locale, and I had to do this because the redirect wasn't working, so this would just introduce another redirect.  Plus, the rest of the repository is in english, and there is a language selector on the GH docs page.

Reply:
https://docs.github.com/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization is a redirect without the `en/free-pro-team@latest/` part.